### PR TITLE
Use pre-commit for linting.

### PR DESCRIPTION
**Pull request by sallner**

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Copyright (c) 2014 gocept gmbh & co. kg
 # See also LICENSE.txt
 
+import gocept.package.sphinxconf
+
+
 _year_started = 2014
 
-import gocept.package.sphinxconf
```
Comment (icemac) [inline doc/conf.py:6]: Could it be that the `_year_started` is already needed by the import?
(I hope there was a reason why we did such a strange order.)

**Review by icemac — APPROVED** [review]
LGTM besides a question.

File: doc/conf.py (hunk 1 above)
Comment (sallner) [reply doc/conf.py:6]: Yes, we do ... We should actually switch away from `gocept.package` ...

Comment (sallner) [reply doc/conf.py:6]: hm, I tried it actually and it worked. We just read in the `conf.py` when we call `set_defaults()`. So I will keep it like this.

**Pull request merged**